src/Raven.Server/Smuggler/Documents/Actions/DatabaseIndexActions.cs

```diff
@@ -38,8 +49,17 @@ public async ValueTask WriteIndexAsync(IndexDefinitionBaseServerSide indexDefini
         await _controller.CreateIndexAsync(indexDefinition, RaftIdGenerator.DontCareId);
     }
 
-    public async ValueTask WriteIndexAsync(IndexDefinition indexDefinition)
+    public async ValueTask WriteIndexAsync(IndexDefinition indexDefinition, AuthorizationStatus authorizationStatus)
     {
+        if (_configuration.Security.AuthenticationEnabled &&
+            authorizationStatus == AuthorizationStatus.ValidUser &&
+            (indexDefinition.Type == IndexType.Map ||
+             indexDefinition.Type == IndexType.MapReduce))
```

Comment:
Why only if this is Map or MapReduce?

Reply:
We don't have the checks about Type in the above method:
https://github.com/ravendb/ravendb/pull/17907/files#diff-bfeecfcaf9cd5a808953831746e8f93b98d1470dd355aff6de46ee6737c66c68R35-R36

Reply:
Here we can get with IndexType.Map ,IndexType.MapReduce and JS index and we don't want to throw for JS indexes

Reply:
Okay. Another question - you're checking `authorizationStatus == AuthorizationStatus.ValidUser`, can `authorizationStatus` be `RestrictedAccess`?

Reply:
No. RestrictedAccess can be set only when we use the endpoint that will block the import completely. ValidUser can be set bt the Certificate type and when we use smuggler directly

Reply:
OK